packages/happn-cluster/lib/services/orchestrator/constants.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  STATES: {
+    CONNECTING: 'CONNECTING',
+    SUBSCRIBING: 'SUBSCRIBING',
+    STABLE: 'STABLE',
+    ISOLATED: 'ISOLATED',
+    CONNECTING_INSUFFICIENT_PEERS: 'CONNECTING-INSUFFICIENT PEERS',
```

Comment:
why is this inconsistent? - and then space

Reply:
Was intended to be 'human readable', should probably have been "CONNECTING - INSUFFICIENT PEERS"
will make it consistent